client/my-sites/plugins/plugin.jsx

```diff
@@ -20,8 +22,9 @@ import HeaderCake from 'components/header-cake';
 import PluginMeta from 'my-sites/plugins/plugin-meta';
 import PluginsStore from 'lib/plugins/store';
 import PluginsLog from 'lib/plugins/log-store';
-import PluginsDataStore from 'lib/plugins/wporg-data/store';
-import PluginsActions from 'lib/plugins/actions';
+import WporgPluginsSelectors from 'state/plugins/wporg/selectors';
+import PluginsActions from 'state/plugins/wporg/actions';
```

Comment:
I don't think we can just remove the `lib/plugins/actions` and replace them with `state/plugins/wporg/actions`. 
I think this might be some sort of typo :/